x/campaign/types/errors.go

```diff
@@ -18,4 +18,6 @@ var (
 	ErrAccountNotFound     = sdkerrors.Register(ModuleName, 8, "account not found")
 	ErrSharesDecrease      = sdkerrors.Register(ModuleName, 9, "shares can't be decreased")
 	ErrVouchersMinting     = sdkerrors.Register(ModuleName, 10, "vouchers can't be minted")
+	ErrInvalidVouchers     = sdkerrors.Register(ModuleName, 11, "invalid vouchers")
+	ErrInsufficientFunds   = sdkerrors.Register(ModuleName, 12, "account with insufficient funds")
```

Comment:
We can precise vouchers here

Reply:
Sorry, I didn't understand. Do you mean the error description?

Reply:
yes